sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,87 @@
+(29001,3,1648.31,-6043.64,127.5836,'High Inquisitor Valroth');
+UPDATE `creature_template` SET `ScriptName`='',`AIName`='SmartAI' WHERE `entry`=29001;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=29001 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(29001,0,0,0,54,0,100,0,0,0,0,0,0,53,1,29001,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - On just summoned - Start WP'),
+(29001,0,1,2,58,0,100,0,0,0,0,0,0,19,256,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - On WP ended - Remove unit flags'),
+(29001,0,2,3,61,0,100,0,0,0,0,0,0,8,2,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - Linked event - Set aggressive react state'),
+(29001,0,3,0,61,0,100,0,0,0,0,0,0,49,0,0,0,0,0,0,21,10,0,0,0,0,0,0,0,'High Inquisitor Valroth - Linked event - Attack start'),
+(29001,0,4,0,0,0,100,0,0,40,3400,4700,0,11,52926,64,0,0,0,0,2,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - IC CMC - Cast \'Valroth''s Smite\''),
+(29001,0,5,6,24,0,100,0,52894,0,7000,12000,0,11,52922,64,0,0,0,0,2,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - On target has \'Anti-Magic Zone\' aura - Cast \'The Inquisitor''s Penance\''),
+(29001,0,6,0,61,0,50,0,0,0,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - Linked event - Say line 0'),
+(29001,0,7,0,2,0,100,1,0,50,0,0,0,11,38210,64,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - Between 0-50% HP - Cast \'Renew\' (No Repeat)'),
+(29001,0,8,9,6,0,100,0,0,0,0,0,0,11,52929,2,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - On just died - Cast \'Summon Valroth''s Remains\''),
+(29001,0,9,0,61,0,100,0,0,0,0,0,0,1,1,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'High Inquisitor Valroth - Linked event - Say line 1');
+
+-- Crimson Acolyte
+DELETE FROM `smart_scripts` WHERE `entryorguid`=29007 AND `source_type`=0 AND `id` IN (3,4,5);
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (2900700,2900701,2900702) AND `source_type`=9;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(29007,0,3,0,54,0,100,0,8,0,0,0,0,88,2900700,2900701,2900702,0,0,0,1,0,0,0,0,0,0,0,0,'Crimson Acolyte - On just summoned - Random action list'),
```

Comment:
The result of this could be the same for all Acolytes, and end up at the same spot on op of each other.

Reply:
https://youtu.be/sIaQkKfHGVE?t=46

Reply:
Ok, so the chase movement will override the final position anyway.